posting/list.go

```diff
@@ -963,12 +962,13 @@ func (l *List) ToBackupPostingList(bl *pb.BackupPostingList, alloc *z.Allocator)
 	// out is only nil when the list's minTs is greater than readTs but readTs
 	// is math.MaxUint64 so that's not possible. Assert that's true.
 	x.AssertTrue(out != nil)
-	// defer out.free()
 
 	ol := out.plist
 	bm := roaring64.New()
-	if err := bm.UnmarshalBinary(ol.Bitmap); err != nil {
-		return nil, errors.Wrapf(err, "failed when unmarshal binary bitmap")
+	if ol.Bitmap != nil {
```

Comment:
Is this nil check necessary?

Reply:
Yes. In case of empty list `ol.Bitmap` turns out to be `nil` and `bm.UnmarshalBinray()` panics.